upload/catalog/view/template/product/category.twig

```diff
@@ -33,15 +33,21 @@
           </div>
         {% else %}
           <div class="row">
-            {% for category in categories|batch((categories|length / 4)|round(1, 'ceil')) %}
-              <div class="col-sm-3">
-                <ul>
+            <div class="col-sm-3">
+              <ul>
+                {% for category in categories) %}
                   {% for child in category %}
                     <li><a href="{{ child.href }}">{{ child.name }}</a></li>
                   {% endfor %}
-                </ul>
-              </div>
-            {% endfor %}
+                  {% if loop.index is divisible by(4) and not loop.last %}
```

Comment:
For some reason, this change is still based on an older way to put it but does make more sense due to it's simplicity to validate this. It would be, then, to know from the PHP TWIG Engine which PHP function would be more optimized to accomplish this.